src/commands/sourcemaps/README.md

```diff
@@ -19,7 +19,7 @@ It is also possible to override the full URL for the intake endpoint by defining
 
 ### `upload`
 
-This command will upload all javascript sourcemaps and their corresponding javascript file to Datadog in order to un-minify front-end stack traces received by Datadog.
+This command will upload all JavaScript sourcemaps and their corresponding JavaScript file to Datadog in order to un-minify front-end stack traces received by Datadog.
```

Comment:
**🥜 nitpick:** ‏ This could be interpreted as if we uploaded the source files, but we only upload the bundles. Maybe rephrase it like other sourcemap upload commands (ex: `react-native upload`):

```suggestion
This command will upload all JavaScript sourcemaps and their corresponding JavaScript bundles to Datadog in order to un-minify front-end stack traces received by Datadog.
```

Reply:
Oh, good nitpick indeed 👍